modules/node/persistence/cms_node_storage_sql.e

```diff
@@ -398,6 +399,13 @@ feature -- Change: Node
 			sql_modify (sql_delete_node_revisions, l_parameters)
 			sql_finalize
 
+				-- we remove node_aliases
```

Comment:
you should use a transaction, and also check for "has_error"
to commit or rollback